Second

Olegsuus opened the pull request.

Olegsuus committed:
Сделал миграции таблиц в ДБ

Olegsuus committed:
1)Добавил структуры user и banner.
2)Добавил функции для получения юзера по токену и баннера по id
3)Добавил комментарии к функциям.

Olegsuus committed:
Сделал вывод ошибок через пакет log

Olegsuus committed:
поменял название типа в GetConfig

RyabininAleck commented on cmd/app/main.go:
```diff
@@ -1,14 +1,14 @@
-//	migration.Migration(cfg)
-//}
+func main() {
+	App := &app.App{}
```
Надо ИНИЦИПАЛИЗИРОВАТЬ ИМЕННО ПОСЛЕ СОЗДАНИЯ db

Olegsuus committed:
GOL-51:
1)Создал интерфейс storage
2)Написал migration для автоматический миграций
3)Создал структуру DataBase
4)Написал методы для этой структуры для получения списка баннеров
5)дописал main
6)Создал структуру для отображение таблицы Tags

Olegsuus committed:
GOL-51:
1)Добавил метод для получения banner по f_id через Join
2)Добавил метод Close() для структуры database
